src/_main_/settings.py

```diff
@@ -135,21 +135,21 @@
 # Database
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
 DATABASES = {
-    'default': {
+    'remote-default': {
         'ENGINE': os.environ.get('DATABASE_ENGINE'),
         'NAME': os.environ.get('DATABASE_NAME'),
         'USER': os.environ.get('DATABASE_USER'),
         'PASSWORD': os.environ.get('DATABASE_PASSWORD'),
         'HOST': os.environ.get('DATABASE_HOST'),
         'PORT': os.environ.get('DATABASE_PORT')
     },
-    'local-default': {
+    'default': {
         'ENGINE': os.environ.get('DATABASE_ENGINE'),
         'NAME': 'postgres2',
         'USER': 'Brad',
         'PASSWORD': '',
         'HOST': 'localhost',
-        'PORT': '5432'
+        'PORT': '5555'
```

Comment:
i would love if this was in the env file so everyone can put their port. And that goes for all the variables above.  
Its probably better if your own local version of `dev.env` sets the db values to these and so the default one relies on that

Reply:
I added IS_LOCAL and a local.env file - same as dev.env but with my database settings.